web/.env.local

```diff
@@ -0,0 +1,7 @@
+REEARTH_WEB_DEVELOPER_MODE=
```

Comment:
```suggestion
```
I think this one is only for show create beta project button on dashboard, which is not exists any more, could you help double confirm whether it is still useful?

Reply:
Looks like this one is used in `e2e/utils/config.ts`

Reply:
I think it's been configed but not in use any more, i removed it.